imap_processing/tests/cdf/test_imap_cdf_manager.py

```diff
@@ -24,6 +26,7 @@ def test_add_instrument_global_attrs():
     assert instrument2_instrument["Project"] == "STP>Solar-Terrestrial Physics"
 
 
+@pytest.mark.xfail(reason="Find out why this is failing")
```

Comment:
This test is run with the non-standard source directory. That means that the new call to `self.load_variable_attributes("imap_constant_attrs.yaml") is failing because it looks for that file in `imap_processing/tests/cdf`. Since this is just testing coverage for manually setting the source dir, I would suggest that mocking out the `load_variable_attributes` function is appropriate.

Reply:
Thank you for that tip! That fixed that test fail.